app/models/ApplicationData.scala

```diff
@@ -36,10 +36,17 @@ object ApplicationData {
   object ApplicationStatus extends Enumeration {
     type ApplicationStatus = Value
     // format: OFF
-    val REGISTERED, CREATED, IN_PROGRESS, SUBMITTED, WITHDRAWN, PHASE1_TESTS,
-    ALLOCATION_CONFIRMED, ALLOCATION_UNCONFIRMED, AWAITING_ALLOCATION, FAILED_TO_ATTEND,
-    ASSESSMENT_SCORES_ENTERED, ASSESSMENT_SCORES_ACCEPTED, AWAITING_ASSESSMENT_CENTRE_RE_EVALUATION, ASSESSMENT_CENTRE_PASSED,
-    ASSESSMENT_CENTRE_FAILED, ASSESSMENT_CENTRE_PASSED_NOTIFIED, ASSESSMENT_CENTRE_FAILED_NOTIFIED = Value
+    val WITHDRAWN, CREATED, IN_PROGRESS, SUBMITTED = Value
+    val PHASE1_TESTS, PHASE1_TESTS_PASSED, PHASE1_TESTS_FAILED = Value
+    val PHASE2_TESTS = Value
+
+    val REGISTERED = Value
+    val IN_PROGRESS_PERSONAL_DETAILS, IN_PROGRESS_SCHEME_PREFERENCES, IN_PROGRESS_PARTNER_GRADUATE_PROGRAMMES, IN_PROGRESS_ASSISTANCE_DETAILS,
+    IN_PROGRESS_QUESTIONNAIRE, IN_PROGRESS_PREVIEW = Value
+    val ONLINE_TEST_FAILED, ONLINE_TEST_FAILED_NOTIFIED, AWAITING_ONLINE_TEST_RE_EVALUATION,
+    AWAITING_ALLOCATION, FAILED_TO_ATTEND, ALLOCATION_UNCONFIRMED, ALLOCATION_CONFIRMED, ASSESSMENT_SCORES_ENTERED,
+    ASSESSMENT_SCORES_ACCEPTED, AWAITING_ASSESSMENT_CENTRE_RE_EVALUATION, ASSESSMENT_CENTRE_PASSED,
```

Comment:
It looks like you've readded some old statuses here, is ONLINE_TEST_FAILED, notified etc a possible state?